main.cpp

```diff
@@ -30,6 +30,10 @@
 #include <iostream>
 #include <vpnes/vpnes.hpp>
 #include <vpnes/gui/gui.hpp>
+#include <vpnes/config/config.hpp>
+
+#include <boost/program_options.hpp>
```

Comment:
Please include headers in this order:
1. C++ C libraries
2. C++ libraries
3. 3rd-party libraries
4. `vpnes` libraries

General headers should come first, specific ones should be at the end